kaocha-runner.el

```diff
@@ -51,9 +51,8 @@
   (cider-nrepl-request:eval
    code
    callback
-   (cider-current-ns)
-   nil nil nil
-   (cider-current-repl nil 'ensure)))
+   nil nil nil nil
```

Comment:
Why did you switch to `nil` here?

Reply:
I think this is unnecessary now as we pass the ns explicitly to kaocha when we run tests for an ns. It also potentially breaks the call in the clojurescript case as the we always talk to clj REPL now where the clojurescript ns might not exist.